etc/upgrades/v28-precompiles/testnet-upgrade-info.yaml

```diff
@@ -0,0 +1,12 @@
0000000000000000000000000000000000000000000000000000000ef3f0bae00000000000000000000000000000000000000000000000000000000f4ff5e2e00000000000000000000000000000000000000000000000000000000f5c1182c00000000000000000000000000000000000000000000000000000000facd743b00000000000000000000000000000000000000000000000000000000fd791f3c000000000000000000000000000000000000000000000000000000000000000000000000000000006e81a994801457a8d257a7fca3ebf4864719d4e7000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000000b042901c70000000000000000000000000000000000000000000000000000000012f43dab00000000000000000000000000000000000000000000000000000000eb67241900000000000000000000000000000000000000000000000000000000263b7f8e000000000000000000000000000000000000000000000000000000006c0960f9000000000000000000000000000000000000000000000000000000007efda2ae00000000000000000000000000000000000000000000000000000000b473318e00000000000000000000000000000000000000000000000000000000d077255100000000000000000000000000000000000000000000000000000000d07b90d100000000000000000000000000000000000000000000000000000000ddcc9eec00000000000000000000000000000000000000000000000000000000e4948f4300000000000000000000000000000000000000000000000000000000000000000000000000000000662b6f997f7d5ab22159cdf1b47a73199f911dbd00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000000040f23da4300000000000000000000000000000000000000000000000000000000e12a61370000000000000000000000000000000000000000000000000000000098f8196200000000000000000000000000000000000000000000000000000000cf02827d0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001e00000000000000000000000009cc4a8412df560f2e77e3d85e14b7727d88cb0430000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000100085f9382a7928dd83bfc529121827b5f29f18b9aa10d18aa68e1be7ddc35010005f72e443c94460f4583fb38ef5d0c5cd9897021c41df840f91465c0392e01000d83e0329d9144ad041430fafcbc2b388e5434db8cb8a96e80157738a1da00000000000000000000000000000000000000000000000000000000044aa200000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000f4240000000000000000000000000000000000000000000000000000000000001d4c00000000000000000000000000000000000000000000000000000000004c4b40000000000000000000000000000000000000000000000000000000000000182b8000000000000000000000000000000000000000000000000000000000ee6b2800000000000000000000000000000000000000000000000000000000000000001"
```

Comment:
To upgrade a chain to v28 one has to use `upgrade_cut_data` (for gateway), but to migrate a chain on top of GW for the first time, one has to use `diamond_cut_data`. What is provided here is the `diamond_cut_data`, I dont think it is the right place for it, here is [where](https://github.com/matter-labs/zksync-era/blob/3cf0b9ca61d221b1913e9dbadc69d318d7441497/etc/env/ecosystems/gateway/stage_gateway.yaml#L6) it is located for stage. 

Please:
- create the corresponding file for testnet_gateway/mainnet_gateway
- ensure to use the upgrade cut data here.
- the name "gateway_diamond_cut" in this file is bad, we should rename it to "gateway_upgrade_diamond_cut" to avoid such errors